Utils/Utils/String/StringBuilder.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  StringBuilder.swift
+//  Utils
+//
+//  Created by Alexander Filimonov on 28/06/2019.
+//  Copyright © 2019 Surf. All rights reserved.
+//
+
+import Foundation
+
+public class StringBuilder {
```

Comment:
I guess, you can leave small documenting comment about a purpose of the class.

Reply:
Yes, made it